src/components/EventCard.tsx

```diff
@@ -51,24 +51,44 @@ const EventCard = ({
     return <EventTag tag={t} />;
   });
 
+  let [regStatus, setRegStatus] = useState(true);
+
+  let regButtonStyle = regStatus ? "button-base-white px-2 my-2 mr-5" : "button-base-red px-2 my-2 mr-5";
```

Comment:
I'd add more padding to the button when it's red so that it doesn't look too jarring when it's switched between Unregister and Register!

```suggestion
  let regButtonStyle = regStatus ? "button-base-white px-2 my-2 mr-5" : "button-base-red px-4 my-2 mr-5";
```

Reply:
Fixed width button? Would look nicer